src/applications/gi/components/vet-tec/VetTecScoContact.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+export const VetTecScoContact = (sco, header) => {
```

Comment:
Can you make sure this is unit tested?

Reply:
Hi @cvalarida I added some additional coverage for `VetTecScoContact` to meet the Code Climate threshold yesterday. Let me know if you have any other suggestions